storage/innobase/xtrabackup/test/t/pxb-1870.sh

```diff
@@ -23,5 +23,5 @@ PARTITION BY HASH (uid)
 PARTITIONS 3000;
 EOF
 
-xtrabackup --backup --lock-ddl=false --target-dir=$topdir/backup
+xtrabackup --backup --target-dir=$topdir/backup
```

Comment:
Good catch. There is no reason to have --lock-ddl=false here.

Reply:
The reason, with --lock-ddl=false, the file close optimization is disabled (implicitly) and then we required high open file limit in operating system.

Reply:
@satya-bodapati in this case, isn't keeping the number of open files high a key part of reproducing the bug PXB-1870 fixes? Perhaps we should keep lock-ddl disabled.

Reply:
@altmannmarcelo PXB-1870 was not about open files, It was SDI buffers and memory allocations.